### PR TITLE
chore(regressions-tests): Update regression-tests.yml

### PR DESCRIPTION
**Pull request by ashotland**
Run every 3 hours
Add -s to pytest invocation

### CODE REVIEW

File: .github/workflows/regression-tests.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
           pip install -r dragonfly/requirements.txt
           export DRAGONFLY_PATH="${GITHUB_WORKSPACE}/build/dragonfly" # used by PyTests
 
-          pytest -xvr dragonfly --ignore=dragonfly/replication_test.py
+          pytest -xvrs dragonfly --ignore=dragonfly/replication_test.py
```
Comment (adiholden) [inline .github/workflows/regression-tests.yml:54]: when s flag is after the r flag this means that the summery printed will show only skipped tests
https://docs.pytest.org/en/6.2.x/usage.html#detailed-summary-report

Comment (ashotland) [reply .github/workflows/regression-tests.yml]: :-o Moved to the beginning and verified locally

**Comment by ashotland** [comment]
Hi @adiholden can you please take another look?

**Review by adiholden — APPROVED** [review]

**Comment by adiholden** [comment]
@ashotland there is a re-request review button at the reviews section. So you can just click it and I will get notified

**Pull request merged**